source/man/doveadm-dump.1.rst.in

```diff
@@ -75,6 +78,10 @@ ARGUMENTS
    only a directory is specified, doveadm tries to detect the type of
    files under it and dumps them.
 
+*type-specific args*
+   - **private_key** = */path* to decrypt file contents with dcrypt-file.
+   - **password** = *secret* for private_key with dcrypt-file.
```

Comment:
This will become a confusing list when there are more than dcrypt-file parameters. Maybe put these into the type-specific text, something like:

```
   **dcrypt-file**
      Dump metadata of a dcrypt encrypted file.
      :dovecotadded: 2.4.0,3.0.0

      Dump contents if ``private_key`` is provided.

      Type-specific arguments: [ *private_key* [ *password* ] ]
      - **private_key** = */path* to decrypt file contents
      - **password** = *secret* for private_key
```

Also while at it, you could document "index" type having optional "uid" parameter to dump only a specific message UID.